doc/Learn/Tutorials/Logging/HowTo-InternalLogging.md

```diff
@@ -5,57 +5,73 @@ uid: Learn.Tutorials.Logging.UseInternalLogging
 
 `Uno.Extensions.Logging` provides logging capabilities tailored to your target platform. It allows the recording of events for XAML layout, Uno-internal messages, and custom events with severity and verbosity levels of your choice.
 
-> [!Tip] 
+> [!IMPORTANT] 
 > This guide assumes your application has already opted into logging. To find out how to do this, refer to the tutorial [here](xref:Learn.Tutorials.Logging.UseLogging)
 
 ## Step-by-steps
 
 ### 1. Enable Uno internal logging
 
-* To log Uno-internal messages, pass `true` for the `Build` method's `enableUnoLogging` argument:
+* To log Uno-internal messages, use `ConnectUnoLogging()` on the built `IHost` instance:
```

Comment:
This isn't required any more - the new pattern already includes this call

Reply:
@nickrandolph Since this step is no longer needed, does that mean `UseLogging()` enables internal logging? What steps should I include now to enable it with the new pattern?